spring-kafka/src/main/java/org/springframework/kafka/retrytopic/EndpointCustomizerFactory.java

```diff
@@ -84,7 +92,17 @@ protected EndpointCustomizer addSuffixesAndMethod(DestinationTopic.Properties pr
 		};
 	}
 
-	protected Collection<EndpointCustomizer.TopicNamesHolder> customizeAndRegisterTopics(
```

Comment:
You cannot remove `public`/` protected` API if you'd like your fix to be back-ported.
This is what called a breaking change.
The point of point release is drop-n-run for the target project, so anything in that project can fail with a new point version.

Reply:
Sure, for some reason I hadn't noticed it was `protected`, thanks.